### DISCUSSION TITLE
Diff Lock and AWD disable at 20MPH

### DISCUSSION THREAD

**Discussion by Bacon117** [Mogli12/KeyboardSteer#824] [General]
Is there any way to keep the diffs locked and AWD enabled above 20MPH? In my farm I have multiple spots that need Diff Lock and AWD but in between I reach above 20MPH, and it necessitates re-enabling the locks.  

I don't mind modifying my copy of the mod, just need to know what to modify. 

Thanks.

**Comment by Mogli12** [comment]
Hi,

Please check if file vehicleControlAddon.xml exists in the savegame folder. If not, you can create it. 
```xml
<?xml version="1.0" encoding="utf-8" standalone="no" ?>
<VCAGlobals>
    <turnOffAWDSpeed     type="float" value="0"/>
</VCAGlobals>
```
The default value is 30; its unit is kph. 

You can copy the file to the modSettings folder. This file is used when creating a new savegame.

**Comment by Bacon117** [comment]
Thanks so much!  You're good people Mogli.
